server/models/Collective.js

```diff
@@ -1555,6 +1556,33 @@ export default function(Sequelize, DataTypes) {
       .then(() => this.getTiers());
   };
 
+  // edit the notifications of this collective (create/update/remove)
```

Comment:
This is a very bad pattern to edit notifications of a collective. I know that you used what we (I) did for editing members, but it's bad. It was a shortcut and it's bug prone (we should also refactor `editMembers` and `editTiers` at some point).
I'd highly recommend to create new mutations to keep things clear.

Also as a paradigm, I don't think it's right to have `editNotifications` linked to `editCollective`. You shouldn't have to be an admin of a collective to register a webhook for events happening to that collective (though it's fine to assume that for a MVP if it makes things simpler). The only thing is that if you are not an admin of the collective, you cannot receive private information about a transaction (typically the email of a user or payment method or attachment to an expense).

Reply:
@xdamman I have created a new mutation for editing notifications.